### PR TITLE
fbf8 sample

### PR DESCRIPTION
**Pull request by mahikaw**
JSON files for FBF8'2017  Sample

### CODE REVIEW

**Comment by mariobehling** [comment]
@mahikaw Please ask and assign your co-developers as reviewers.

**Comment by mariobehling** [comment]
@mahikaw This PR still does not follow best practices. There is no issue and no-one was asked to review. What is going on?

**Comment by mahikaw** [comment]
@mariobehling I am really sorry I somehow skipped the previous comment. I have been busy with the generator. I will be sending samples for both my PRs today as I have been able to set up the generator yesterday only.
@iamareebjamal @arp95 @niranjan94 could you please review this.

File: sample/F8/tracks [hunk 1]
```diff
@@ -0,0 +1,117 @@
+[
+{
+        "id": 1, 
+        "name": “ADS & MONETIZATION”,
```
Comment (arp95) [inline sample/F8/tracks]: only one track?
please add more.

File: sample/F8/sessions [hunk 2]
```diff
@@ -0,0 +1,1249 @@
...
+         "comments": "", 
+        "language": null, 
+        "slides": null, 
+        "audio": null, 
+        "video": null, 
+        "signup_url": null, 
+        "state": "accepted", 
+        "timezone": "UTC",
+        "session_type": {}, 
+        "track": {},
```
Comment (arp95) [inline sample/F8/sessions]: add the track to which the session belongs. this is important.

File: sample/F8/sessions [hunk 3]
```diff
@@ -0,0 +1,1249 @@
...
+        "end_time": "2017-04-18T12:30:00.000Z"
+
+    }, {
+        "short_abstract": null,
+        "long_abstract": "null",
+        "id": 5,
+        "subtitle": "introduction-to-the-camera-effects-platform",
+        "title": "Introduction to the Camera Effects Platform",
+
+        "sessionDescription": "Come hear more about Camera Effects Platform, a suite of creative tools that lets you create effects for the Facebook camera, from simple photo frames to interactive masks and effects that use Augmented Reality (AR). In this session you’ll learn what camera effects are, how to create and submit your own, and how you can connect with the audiences that matter to you by providing rich experiences that help them express themselves.",
```
Comment (arp95) [inline sample/F8/sessions]: why create a new attribute when we have "short_abstract" and "long_abstract" for gathering the information of the event?

File: sample/F8/sessions [hunk 4]
```diff
@@ -0,0 +1,1249 @@
...
+            "id": 7,
+            "name": "Andrew Mo"
+        }, {
+            "organisation": "Facebook",
+            "id": 8,
+            "name": "Mike Armstrong"
+        }],
+
+
+        "tags": ["Virtual Reality"]
```
Comment (arp95) [inline sample/F8/sessions]: this is a good feature. but i think then we would need a tags.json file as well to collect all the tags we have. lets see what others say on this

**Comment by niranjan94** [comment]
@mahikaw I keep telling you not to close and open new PRs like this for the same issue. We loose the conversations linked with the previous PR.

**Comment by mahikaw** [comment]
@niranjan94 my reason is same as before, I had re-forked the repo and so I don't have access to the earlier branches. This is the last such case and it won't happen again.

**Comment by niranjan94** [comment]
Why do you keep re-forking your repo ? There's no need to.

**Comment by niranjan94** [comment]
You can add the upstream repo (that is the fossasia repo) as a remote and update your fork by fetching and fast-forwarding your branch.

**Comment by mahikaw** [comment]
@niranjan94 I did it just once and so I lost the both the branches related to the repos. I sincerely apologise for any inconvenience caused. Won't repeat this.

**Comment by arp95** [comment]
@mahikaw please fork the repo once. then create new local branches and update as pointed above from an upstream repo.

**Comment by mahikaw** [comment]
Yes, I am doing that. Thanks for the information @arp95 @niranjan94

**Pull request closed without merging**